commons/commons-json-provider/src/main/java/com/clicktravel/common/http/application/ObjectMapperProvider.java

```diff
@@ -37,13 +33,12 @@
     public ObjectMapperProvider() {
         objectMapper = new ObjectMapper();
         objectMapper.registerModule(new JodaModule());
-        objectMapper.setAnnotationIntrospector(AnnotationIntrospector.pair(new JacksonAnnotationIntrospector(),
-                new JaxbAnnotationIntrospector(TypeFactory.defaultInstance())));
         objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
         objectMapper.configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
         objectMapper.setSerializationInclusion(Include.NON_NULL);
 
-        // TODO Uncomment following code when completing migration of JSON provider to Jackson
+        // Swagger codegen generates the following code, but it seems to be redundant
+        // as the Java model files for enums have @JsonValue on the toString() methods
         // objectMapper.enable(SerializationFeature.WRITE_ENUMS_USING_TO_STRING);
         // objectMapper.enable(DeserializationFeature.READ_ENUMS_USING_TO_STRING);
```

Comment:
I wanted to retain this information, as it may help us in future when we move to using Swagger codegen for generating clients